app/src/Components/AdminPage/VisitorsTable/VisitorsTable.js

```diff
@@ -31,9 +30,19 @@ class VisitorsTable extends React.Component {
         })
         .then(data=>data.json())
         .then((data)=>{
+            if (data.message === null || data.message ==='Invalid token' || data.message === 'Malformed token' ||
+            data.message === 'Token has expired' || data.message === 'Token error' ||
```

Comment:
purely opinion: as its checking 6 cases, maybe it might read easier having 2 per-line. For example: 

            if (data.message === null || data.message ==='Invalid token' || 
                data.message === 'Malformed token' || data.message === 'Token has expired' || 
                data.message === 'Token error' || data.message === 'No token received') {
                localStorage.removeItem('bearerToken');
                window.location.replace(this.state.appUrl)
            }

Reply:
Like it. Done.